scss/_buttons.scss

```diff
@@ -121,6 +123,13 @@
 // scss-docs-start btn-variant-loops
 @each $color, $value in $theme-colors {
   .btn-#{$color} {
+
+    :not(.btn-check) + &:active,
```

Comment:
I'm not sure what repurcussions this might have but my logic is based on the fact that this css variable is only ever defined in the button variants per:

https://github.com/twbs/bootstrap/blob/main/scss/mixins/_buttons.scss#L14
and
https://github.com/twbs/bootstrap/blob/main/scss/mixins/_buttons.scss#L29

Reply:
Some screenshots of manual test (I'm new to codebase so I just hacked the docs site):

Test case is to only have `.btn` class + a custom class that defines a background:

<img width="1904" alt="image" src="https://user-images.githubusercontent.com/142403/194652319-29ca4fbf-e289-4949-89f8-3ac2f60093a3.png">

Test that variants `:active` still kick-in:
<img width="1904" alt="image" src="https://user-images.githubusercontent.com/142403/194652421-1318733a-f912-4255-99c5-d0223207b107.png">

Reply:
Sorry, noob to codebase but running `test`:
<img width="790" alt="image" src="https://user-images.githubusercontent.com/142403/194652609-0fec39a6-3ead-4948-81f8-7e19bb953ff9.png">

Reply:
> Sorry, noob to codebase but running npm run testshows this -- this indicates no breakage correct?

Yes if you don't see any red messages and that the process exits without any errors, there's no breakage. However, there are very few tests regarding the buttons. Most of them, when modifying CSS, must be done manually.

Reply:
Any suggestions on what to do next here?